test/proposals.functional.tests.js

```diff
@@ -0,0 +1,52 @@
+var BridgeTestable= artifacts.require("./BridgeTestable");
+var ProposalsContract = artifacts.require("./ProposalsContract");
+var PreserveBalancesOnTransferToken = artifacts.require("./PreserveBalancesOnTransferToken");
+var util = require('ethereumjs-util');
+const time = require('./helpers/time');
+
+require('chai')
+	.use(require('chai-as-promised'))
+	.use(require('chai-bignumber')(web3.BigNumber))
+	.should();
+
+contract('ProposalsContract', (accounts) => {
+	const creator = accounts[0];
+	const u1 = accounts[1];
+	const u2 = accounts[2];
+	const u3 = accounts[3];
+	const u4 = accounts[4];
+	const u5 = accounts[5];
+
+	var bridgeTestable;
+	var proposalsContract;
+	var preserveBalancesOnTransferToken;
+
+	describe('Positive scenario I', function(){
```

Comment:
should we also check negative scenarios? For example, that proposal is not finalizable before `daysToVote` pass